src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -124,6 +129,12 @@ public static Angle heightToRotations(Distance height) {
         .times(Rotations.one());
   }
 
+  public Angle ChineseRemander() {
```

Comment:
You need to use the method to seed the elevator position somewhere. Maybe rewrite the zeroEncoder() command to use this?